pkg/lib/editor/editor.go

```diff
@@ -111,11 +121,49 @@ func RunConfigEditor(password, configPath, operatorEndpoint string, readOnlyFiel
 		fs.ServeHTTP(w, r)
 	})
 
+	// Create server base
 	s := &http.Server{
 		Addr:    ":" + strconv.Itoa(opts.port),
```

Comment:
should take port as a string, its even in the c libs it is used as a string.

Reply:
it does take it as a string, but since i don't specify a hostname i need to put the ":8080"

Reply:
Why are you performing a string conversation from integer to string then?

Reply:
good point, was reading port as in int from cli. fixed.